addOns/authhelper/src/test/java/org/zaproxy/addon/authhelper/AuthUtilsUnitTest.java

```diff
@@ -428,10 +428,11 @@ void shouldDefaultToNoTokens() throws Exception {
         HttpMessage msg =
                 new HttpMessage(
                         new HttpRequestHeader(
-                                "GET / HTTP/1.1\r\n"
-                                        + "Header1: Value1\r\n"
-                                        + "Header2: Value2\r\n"
-                                        + "Host: example.com\r\n\r\n"),
+                                """
+                                        GET / HTTP/1.1\r
+                                                Header1: Value1\r
+                                                Header2: Value2\r
+                                                Host: example.com\r\n\r\n"""),
```

Comment:
This should be properly indented and throughout.

Reply:
456, 499, 536, 561. In HeaderBasedSessionManagementMethodTypeUnitTest` 144. In`VerificationRequestDetailsUnitTest` 81, 114, 150, 302.

Reply:
OH sorry when you said "throughout" I thought just the class

Reply:
Ugh sorry I only looked in the class and primarily for requests

Reply:
I'm not sure what to do at 499 because that isn't valid JSON, I've out-dented things a bunch so hopefully that's good. But I kinda question the test now that it seems to be invalid content.

Reply:
The escapes should not be added though. (The parser we are using does not care if it's using single or double.)

Reply:
Oh okay. I had thrown it into another tool and it was unhappy with the apostrophes.

Would you rather they were back or that this simply isn't escaped?

Reply:
The latter, there's not need to escape if we are using text blocks.